cmd/main.go

```diff
@@ -42,23 +42,6 @@ var (
 )
 
 func runProbe(ctx context.Context, csiConn connection.CSIConnection) error {
-
-	// Get CSI driver name.
-	glog.Infof("Calling CSI driver to discover driver name.")
-	csiDriverName, err := csiConn.GetDriverName(ctx)
```

Comment:
Can we leave `GetDriverName` part as it is as it should be retrived from `GetPluginInfo()` which is RPC from identity service?

Reply:
Yep, restored..